libbpf-rs/src/program.rs

```diff
@@ -119,6 +119,12 @@ impl OpenProgram {
         }
     }
 
+    /// Set the log level for the bpf program
+    pub fn set_log_level(&mut self, log_level: u32) -> Result<()> {
```

Comment:
Can we document what `log_level` is? It's not obvious to users. Seems to be some bitmap kind of thing, is that right?